### ISSUE TITLE
Allow spaces in rgb values

### ISSUE THREAD

**Issue by alex-wilmer** [ActionIQ/style-builder#2]
Not a huge problem, but `border: 1px solid rgb(0, 0, 0)` throws warning because style-builder splits on whitespace and looks for less than three values. I can submit a PR for it if you want.

**Comment by ghost** [comment]
@alex-wilmer yeah! if you've got a chance to write up a pr that'd be great, otherwise I can probably get to fixing that this week.

**Comment by alex-wilmer** [comment]
https://github.com/ActionIQ/style-builder/pull/3
